doc/7/protocols/websocket/constructor/index.md

```diff
@@ -30,8 +30,9 @@ WebSocket protocol connection options.
 | Property            | Type<br/>(default)               | Description                                                                                  |
 | ------------------- | -------------------------------- | -------------------------------------------------------------------------------------------- |
 | `autoReconnect`     | <pre>boolean</pre><br/>(`true`)  | Automatically reconnect to kuzzle after a `disconnected` event                               |
-| `port`              | <pre>number</pre><br/>(`7512`)   | Kuzzle server port                                                                           |
 | `headers`           | <pre>object</pre>(`{}`)          | Connection custom HTTP headers (e.g. origin, subprotocols, ...)<br/>**(Not supported by browsers)** |
+| `pingInterval` | <pre>number</pre><br/>(`30000`)   | Number of milliseconds between two pings                                         |
```

Comment:
I think `30s` is too high. On a mobile application you want to quickly know if you lost the connection so the SDK can start queuing requests.

The `2s` delay used in templates seems good to me.

Reply:
I didn't assume that the primary use was for a mobile application.
If this is the case, the option remains configurable.
On the other hand, if it's a mobile application, it means more network activity, and therefore more battery consumption.
I'll go for something like 10 seconds by default. 
Wdyt? @scottinet  @Aschen  ?

Reply:
dkdc :expressionless: